src/client/styles/modules/topnav-layout.less

```diff
@@ -63,3 +63,44 @@
     }
   }
 }
+
+div[data-dir='rtl'] .topnav-layout {
+  .left {
+    right: 0;
+    padding-right: 16px;
+    padding-left: 0px;
+  }
+
+  .right {
+    right: unset;
+    left: 0;
+    margin-right: 0px;
+    margin-left: 16px;
+  }
+
+  .center {
+    @media @small {
+      width: calc(100% ~'-' (@left-width + @right-width + 2 * @margin));
+      position: static;
```

Comment:
same with `position` and `display` values, they are the same as above, I think you can probably remove them and only use styles that are being changed.